glib/tests/log.rs

```diff
@@ -1,6 +1,10 @@
 use std::sync::{Arc, Mutex};
 
-use glib::*;
+use glib::{
+    clone, g_critical, g_debug, g_info, g_message, g_warning, log_remove_handler,
+    log_set_default_handler, log_set_fatal_mask, log_set_handler, log_unset_default_handler,
+    LogLevel, LogLevels,
+};
```

Comment:
Why this change?

Reply:
being the macro name `test` importing `glib::*` was overriding """std""" test.

Reply:
anyway, renaming the macro `test -> async_test` solves this